backend/queue-service/src/main/kotlin/com/ticketqueue/queue/service/QueueService.kt

```
@@ -147,6 +148,39 @@ class QueueService(
+    fun leaveQueue(userId: UUID, scheduleId: UUID): QueueDto.LeaveResponse {
+        val keys = listOf(
+            QueueRedisKeys.queue(scheduleId),
+            QueueRedisKeys.active(userId),
+            QueueRedisKeys.userToken(userId, scheduleId)
+        )
+        val args = arrayOf(userId.toString(), scheduleId.toString())
+
+        val result = executeLuaOrThrow(
+            queueLeaveScript, keys, *args,
+            logContext = "leave: scheduleId=$scheduleId, userId=$userId"
+        )
```

detekt (1.23.8)

[warning] In most cases using a spread operator causes a full copy of the array to be created before calling a method. This may result in a performance penalty.

(detekt.performance.SpreadOperator)